src/vs/workbench/contrib/debug/browser/debugSession.ts

```diff
@@ -55,6 +55,10 @@ export class DebugSession implements IDebugSession {
 
 	private readonly _onDidChangeREPLElements = new Emitter<void>();
 
+	private name: string | undefined;
+	private readonly _onDidChangeName = new Emitter<string>();
```

Comment:
I do not understand why do we need `onDidChangeName`?
Isn't `onDidChangeLabel` enough? Or the other way around - point being I think we can survive with only one.